libs/mve/bundle_io.cc

```diff
@@ -480,4 +489,77 @@ save_photosynther_bundle (Bundle::ConstPtr bundle, std::string const& filename)
     out.close();
 }
 
+void
+save_nvm_bundle (Bundle::ConstPtr bundle, const std::vector<NVMCameraInfo> &camera_info, std::string const& filename)
```

Comment:
`Bundle::Ptr
load_nvm_bundle (std::string const& filename,
    std::vector<NVMCameraInfo>* camera_info = nullptr);`

Passing a barely useful CameraInfo is ugly here as well. It's obvious, that I need all the data you created during the import for the export.